package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "troubleshoot",
```

Comment:
Do we need to make troubleshoot an NPM package i.e creating `package.json` & `package-lock.json`? From looking at the implementation it should be sufficient to just run `npm install gaze-run-interrupt` them call `make watchrsync`